docs/codeql/reusables/supported-versions-compilers.rst

```diff
@@ -37,4 +38,5 @@
     .. [7] JSX and Flow code, YAML, JSON, HTML, and XML files may also be analyzed with JavaScript files.
     .. [8] The extractor requires Python 3 to run. To analyze Python 2.7 you should install both versions of Python.
     .. [9] Requires glibc 2.17.
-    .. [10] TypeScript analysis is performed by running the JavaScript extractor with TypeScript enabled. This is the default.
+    .. [10] Swift support is currently in beta. Windows is not supported. Linux support is partial (currently only works with Swift 5.7.3).
```

Comment:
I think it would be clearer to express this positively.

```suggestion
    .. [10] Swift support is currently in beta. Support for the analysis of Swift 5.4-5.7 requires macOS. Swift 5.7.3 can also be analyzed using Linux.
```